.github/workflows/test.yaml

```diff
@@ -33,11 +33,11 @@ jobs:
           ${{ runner.os }}-pip-
     - name: Install requirements
       run: pip install -r requirements_for_test.txt
-    - name: Run tests
-      run: make test
+    # - name: Run tests
+    #   run: make test
```

Comment:
Should this still be commented out? This is preventing the tests from running in CI...

Reply:
No definitely not! My bad, good eye.

Reply:
Now the real question is... do any of the tests actually pass? 😢